troubleshooting.md

```diff
@@ -437,6 +437,8 @@ Failed to get D-Bus connection: Operation not permitted
 Error: non zero exit code: 1: OCI runtime error
 ```
 
+This means that you cannot run systemd inside a CentOS 7 or RHEL 7 container if your host uses cgroups v2. RHEL 8 and Fedora use cgroups v2.
```

Comment:
Fedora 31 and later

Reply:
RHEL 8 does not default to cgroups V2.  It defaults to cgroups V1.